index.html

```diff
@@ -370,7 +371,7 @@ <h2>
       </p>
       <p>
         For a given {{Document}}, the <a>current posture</a> is derived from
-        the current hinge angle and the <a>current screen orientation</a>, and
+        the current hinge angle and the current {{ScreenOrientation}}, and
```

Comment:
Referencing `ScreenOrientation` as an interface sounds too vague. I think it would make more sense to refer to either https://w3c.github.io/screen-orientation/#dfn-current-orientation-type or https://w3c.github.io/screen-orientation/#dfn-screen-orientations. They don't seem to be exported, so you'd need to reference it from here somehow or send a PR to the screen-orientation spec exporting the definition(s) you need first.